docs/posts/gallery.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Gallery
+subtitle: Features & examples
+layout: default
+---
+
+# Features
```

Comment:
**Update:** 
@sidneymbell @colinmegill Here is a link to my proposal for the Gallery GIFs:
https://docs.google.com/document/d/1L9PFv1M7TjiSiP_6yQKJzX10OEwmJye0tdDMUcqiTWU/edit?usp=sharing